README.md

```diff
@@ -348,6 +350,21 @@ References:
 
 </details>
 
+### 2.3 Cleanup configuration files
+
+✅ **Do:**
+Cleanup configuration files when the CLI application gets uninstalled. Optionally, CLI applications can prompt your users to keep the configuration files to skip the re-initialising phase on the next installation for a better user experience.
+
+❌ **Otherwise:**
```

Comment:
Is that okay to keep this section empty?

Reply:
@m-sureshraj we should definitely add this. I didn't notice it was empty before.
Can you please update? I would say that we want to keep the file system clean of any remaining configuration or identifiable data that the CLI tool might have left in when being installed on the system.